labels.mss

```diff
@@ -1,22 +1,29 @@
-#country[zoom>=3][zoom<=7] {
+#country[importance<=3][zoom>=3][zoom<=10],
+#country[importance<=4][zoom>=4][zoom<=10],
+#country[zoom>=5][zoom<=10] {
   text-name: '[name]';
   text-face-name: @bold;
   text-placement: point;
   text-fill: @country_text;
-  text-size: 14;
+  text-size: 12;
   text-halo-fill: @halo;
   text-halo-radius: 1;
   text-wrap-width: 40;
   text-label-position-tolerance: 20;
   text-character-spacing: 1;
   text-placement-type: simple;
   text-placements: 'N,S';
-  text-dx: 10;
-  text-dy: 10;
-  text-margin: 5;
   text-line-spacing: 0.1;
+  [zoom>=5] {
+    text-size: 14;
+    text-dx: 10;
```

Comment:
I guess you removed default text-margin for "diplomatic" reasons? Because it ends with labels being very close, and thus the map being less readable. Just to be sure we *need* this "regression" ;)

![screenshot-20180213180123-574x410](https://user-images.githubusercontent.com/146023/36162872-eeef3636-10e7-11e8-9382-fc7b4cea8d91.png)

Can't we go with a more readable map and have this kind of adjustment if some complain?

Reply:
Mmmh just bad cut / paste for the text-dx and text-dy

Sorry, I agree completely, I even increased it a bit at small zoom levels

Reply:
👍